pkg/routeagent/controllers/route/driver.go

```diff
@@ -3,32 +3,57 @@ package route
 import (
 	"fmt"
 	"io/ioutil"
+	"net"
 
 	"github.com/vishvananda/netlink"
 	"k8s.io/klog"
+
+	"github.com/submariner-io/submariner/pkg/util"
 )
 
-const FlannelInterface = "flannel.1"
+func interfaceExists(iface string) bool {
+	_, err := netlink.LinkByName(iface)
+	return err == nil
```

Comment:
It seems `LinkByName` could return an err for reasons other than not found. Perhaps we should specifically check for `netlink.LinkNotFoundError` and also propagate any other error...